packages/cli/src/commands/__tests__/cli-commands.test.ts

```
@@ -133,9 +134,14 @@ describe("M3 CLI command IA", () => {
     const workflow = "inline-workflow";
     const runtimeDefineSpy = vi.spyOn(oboraSdk.OboraRuntime.prototype, "define");
 
-    await runRun(workflow, {});
+    try {
+      await runRun(workflow, {});
+    } finally {
+      if (prevProvider === undefined) delete process.env.OBORA_LLM_PROVIDER; else process.env.OBORA_LLM_PROVIDER = prevProvider;
```

ESLint

[error] Insert `⏎·····`

(prettier/prettier)